packages/fondue/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -50,6 +51,7 @@ export const RichTextEditor = ({
     border = true,
     toolbarWidth,
     hideExternalFloatingModals,
+    placeholderOpacity = 0.333,
```

Comment:
why do we need this level of control? idk why this is being added but I assume we should at most have something like 

```suggestion
    transparency = 'medium', // medium, low, high
```

Reply:
Well, there is a bit of a trial and error going on trying to make it compatible with different color schemes. I figured it will be easier to give full control, and then we don't need to touch fondue if we decide to change.

Right now we want it to be `0.7` instead of the default `0.333`, but since RTE is used other places as well, not just guidelines, I did not want to just override directly.

I can add the low/medium/high options if you worry it's too much freedom.

Reply:
Gotcha, added